Makefile

```diff
@@ -92,6 +93,8 @@ $(OUT)/trivialDAO_storage.tz: src/**
           } \
         ; config_data = \
           { voting_period = { length = $(voting_period) } \
+          ; proposal_flush_time = ($(voting_period) * 2n) \
+          ; proposal_expired_time = ($(voting_period) * 3n) \
```

Comment:
Here I am not sure if we should hard code `2n` and `3n`, and instead may be we should allow user to specify them, and also  probably check that flush time is greater then 2 * voting_period, and expiry time is greater than flush time.

Reply:
I've added arguments for these fields now.